.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
```

Comment:
I think we would have faster tests if running travis without sudo nor docker and using the postgres service.